src/test/resources/features/acceptance/address-lookup.feature

```diff
@@ -54,23 +54,27 @@ Feature: Select address
     Given I have entered my details up to the postcode page
     When I enter <postcode> as my postcode
     Then I am informed that the postcode is in the wrong format
+    And I see the postcode: <postcode> in the input in the same format I entered it in
 
     Examples:
       | postcode |
       | AA1122BB |
       | A        |
       | 11AA21   |
+      | 11A  A21 |
```

Comment:
Can we have a postcode in lowercase too?

Reply:
Added postcode with lowercase text.